beacon_node/network/src/attestation_service/mod.rs

```diff
@@ -186,18 +186,49 @@ impl<T: BeaconChainTypes> AttestationService<T> {
     pub fn validator_subscriptions(
         &mut self,
         subscriptions: Vec<ValidatorSubscription>,
-    ) -> Result<(), ()> {
+    ) -> Result<(), String> {
+        let current_slot = self
+            .beacon_chain
+            .slot()
+            .map_err(|e| format!("Failed to get current slot: {:?}", e))?;
+
+        let mut state = self
+            .beacon_chain
+            .state_at_slot(current_slot, StateSkipConfig::WithoutStateRoots)
+            .map_err(|e| format!("Failed to get beacon state: {:?}", e))?;
+
+        // Ensure that the committee caches are built
+        state
+            .build_all_committee_caches(&T::EthSpec::default_spec())
```

Comment:
Separate note: we should use the `ChainSpec` from the `BeaconChain` instead of the default spec, in case it has been modified.